src/Graphics/OpenGLContext/mupen64plus/mupen64plus_DisplayWindow.cpp

```diff
@@ -205,9 +186,9 @@ void DisplayWindowMupen64plus::_readScreen2(void * _dest, int * _width, int * _h
 		glReadBuffer(GL_BACK);
 	glReadPixels(0, m_heightOffset, m_screenWidth, m_screenHeight, GL_RGBA, GL_UNSIGNED_BYTE, pBufferData);
 	glReadBuffer(oldMode);
-#else
+	} else {
 	glReadPixels(0, m_heightOffset, m_screenWidth, m_screenHeight, GL_RGBA, GL_UNSIGNED_BYTE, pBufferData);
-#endif
+	}
```

Comment:
BTW, is it possible that that function is called with _front != 0?
If _front is always zero, all that code is redundant and we can left only glReadPixels